cfme/automate/explorer/method.py

```diff
@@ -484,6 +507,15 @@ def create(
                 'playbook_input_parameters': playbook_input_parameters
             })
             validate = False
+
+        if location in ['Ansible Tower Workflow Template', 'Ansible Tower Job Template']:
+            add_page.fill({
+                'method_name': name,
+                'method_display_name': display_name,
+                'ansible_max_ttl': max_ttl,
+            })
+            validate = False
```

Comment:
Why are you setting validate=false here ?
Don't we need to validate after fillng data ?

Reply:
@sshveta For method types such as ```'Ansible Tower Workflow Template', 'Ansible Tower Job Template', 'playbook'```; we don't have such feature with button **validate**. This is applicable only for **inline** type of method.